src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicPreferStreamAsyncMemoryOverloads.Fixer.vb

```diff
@@ -99,5 +133,46 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
 
         End Function
 
+        Protected Overrides Function GetNodeWithNullability(invocation As IInvocationOperation) As SyntaxNode
+
+            Return invocation.Instance.Syntax
```

Comment:
VB does not seem to have nullability, so all the new methods added to this file simply implement the abstract one and make sure to return an expression `SyntaxNode`, since it is required by the fixer.